cabal.project

```diff
@@ -1 +1,2 @@
 packages: ./jl4
+          ./web-service
```

Comment:
since I'm also writing a web service, please make this name more speaking. I currently have 
`jl4-websessions`.

Reply:
Is it better now? :laughing:

Reply:
much appreciate it